src/emitter.ts

```diff
@@ -1052,11 +1054,14 @@ export function emitWebIDl(webidl: Browser.WebIdl, flavor: Flavor) {
                 }
                 return i.iterator.type.map(convertDomTypeToTsType);
             }
-            else if (i.name !== "Window" && i.properties) {
+            else if (i.name !== "Window") {
                 const iterableGetter = findIterableGetter();
-                const lengthProperty = mapToArray(i.properties.property).find(findLengthProperty);
+                const lengthProperty = findLengthProperty(i) || findLengthProperty(allInterfacesMap[i.extends]);
```

Comment:
should this be a recursive function to repeat on every extended type?

Reply:
I'm not sure, as this really is only for `HTMLCollectionOf`. All other interfaces are fine without inheritance check. Should recursion be preferred?